src/api/forms/FormsAPI.js

```diff
@@ -158,7 +159,8 @@ export default class FormsAPI extends EventEmitter {
                     key = property.join('.');
                 }
 
-                changes[key] = data.value;
+                _.set(changes, `${key}`, data.value);
```

Comment:
Why does `key` need to be wrapped in a string literal?

Reply:
It doesn't :D